ert_shared/storage/model.py

```diff
@@ -237,12 +245,49 @@ def __repr__(self):
             self.stds_ref,
         )
 
+    def add_attribute(self, attribute, value):
+        self.attributes[attribute] = Value(value)
+
+    def get_attribute(self, attribute):
+        return self.attributes[attribute].value
+
 
 Observation.response_definitions = relationship(
     "ResponseDefinition", order_by=ResponseDefinition.id, back_populates="observation"
 )
 
 
+class ObservationsAttribute(Entities):
+    __tablename__ = "observations_attribute"
+
+    observation_id = Column(Integer, ForeignKey("observations.id"), primary_key=True)
+    attribute = Column(String)
+    value_id = Column(Integer, ForeignKey("value.id"), primary_key=True)
+    value = relationship("Value")
+
+    observation = relationship(
+        Observation,
+        backref=backref(
+            "observations_attributes",
+            collection_class=attribute_mapped_collection("attribute"),
+            cascade="all, delete-orphan",
+        ),
+    )
+
+
+class Value(Entities):
```

Comment:
If this is only used by observation_attributes, is it maybe a bit too generically named?